application/clicommands/ListCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Icinga\Module\Reporting\Clicommands;
+
+use Icinga\Module\Reporting\Cli\Command;
+use InvalidArgumentException;
+use ipl\Sql\Select;
+
+class ListCommand extends Command
+{
+    /**
+     * List reports
+     *
+     *  USAGE:
+     *
+     *   icingacli reporting list --sort id|name|author --direction ASC|DESC (default DESC)
```

Comment:
Since both params are optional please change the docs as follows.
```suggestion
     * USAGE:
     *
     *   icingacli reporting list [OPTIONS]
     *
     * OPTIONS:
     *
     *   --sort=<id|name|author>    Sort the reports by the given column. Default (name)
     *    --direction=<ASC|DESC>    Sort the reports by the specified sort column in ascending or descending order.
     *                              Default (DESC)
```